docs/writing-docs/autodocs.md

```diff
@@ -66,11 +66,7 @@ To replace the default documentation template used by Storybook, you can extend
 
 <!-- prettier-ignore-end -->
 
-<div class="aside">
-
-💡 Internally, Storybook uses a similar implementation to generate the default template. See the Doc Blocks [API reference](./doc-blocks.md#available-blocks) to learn more about how Doc Blocks work.
-
-</div>
+<YouTubeCallout id="q8SY4yyNE6Q" title="Custom Autodocs with Storybook 7 Docs Page | Quick Tips" />
```

Comment:
When we include these at the page level, we place them immediately under the page title. The thinking there being that videos offer an alternative learning path, and we want to present that as early as possible.

I think we should apply the same pattern here, and place the YouTubeCallout immediately after the section heading.

Reply:
I know, and I thought of it as well, but I'm aware that users might want to get into the video upfront, but when taking a second look at it, I placed it there to give users the code first, and after it the video and the text so that they follow along what's being presented. More than ok with moving it up and seeing how it fares. If need be we can shift it around later on.

Sounds good?